lib/src/issue_table.dart

```diff
@@ -0,0 +1,229 @@
+                alignment: Alignment.topLeft,
+                renderFunction: (context, issue, out) {
+                  final reviewers = (issue.assignees ?? [])
+                      .map((reviewer) =>
+                          formatUsername(reviewer, widget.googlers))
+                      .join(', ');
+                  // TODO: Consider using a RichText widget here.
+                  return ClipRect(
+                    child: Wrap(
+                      children: [
+                        if (reviewers.isNotEmpty)
+                          Text(reviewers, style: rowStyle(issue)),
+                      ],
+                    ),
+                  );
+                },
+                styleFunction: rowStyle,
+                validators: [
+                  (pr) => needsReviewersValidator(widget.googlerUsers, pr),
+                ],
+              ),
+              VTableColumn(
+                label: 'Labels',
+                width: 120,
+                grow: 0.8,
+                alignment: Alignment.topLeft,
+                transformFunction: (issue) =>
+                    (issue.labels).map((e) => "'${e.name}'").join(', '),
+                renderFunction:
+                    (BuildContext context, Issue issue, String out) {
+                  return ClipRect(
+                    child: Wrap(
+                      spacing: 4,
+                      runSpacing: 4,
+                      children: (issue.labels).map(LabelWidget.new).toList(),
+                    ),
+                  );
+                },
+              ),
+              VTableColumn(
+                label: 'Upvotes',
+                width: 50,
+                grow: 0.2,
+                alignment: Alignment.topLeft,
+                transformFunction: (issue) => issue.upvotes.toString(),
+                compareFunction: (a, b) => a.upvotes.compareTo(b.upvotes),
+              ),
+            ],
+          );
+        },
+      ),
+    );
+  }
+}
+
+const TextStyle draftPrStyle = TextStyle(color: Colors.grey);
+
+TextStyle? rowStyle(Issue issue) {
+  if (issue.draft == true) return draftPrStyle;
+
+  return null;
+}
+
+class LabelWidget extends StatelessWidget {
+  final IssueLabel label;
+
+  const LabelWidget(
+    this.label, {
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final chipColor = Color(int.parse('FF${label.color}', radix: 16));
+
+    return Material(
+      color: chipColor,
+      shape: const StadiumBorder(),
+      elevation: 2,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 6.0),
+        child: Text(
+          label.name,
+          style: TextStyle(
+            color: isLightColor(chipColor)
+                ? Colors.grey.shade900
+                : Colors.grey.shade100,
+          ),
+          maxLines: 1,
+          overflow: TextOverflow.ellipsis,
+        ),
+      ),
+    );
+  }
+}
+
+ValidationResult? oldPrValidator(Issue issue) {
```

Comment:
We should update this validator to reflect that it's now working on issues (i.e., updating the function name and the validation messages).